app/javascript/components/pages/task-list/container.spec.js

```diff
@@ -0,0 +1,509 @@
+                  id_search: false
+                }),
+                ListHeaderRecord({
+                  name: "due_date",
+                  field_name: "due_date",
+                  id_search: false
+                }),
+                ListHeaderRecord({
+                  name: "status",
+                  field_name: "status",
+                  id_search: false
+                })
+              ]
+            }
+          },
+          forms: {
+            formSections: OrderedMap({
+              1: FormSectionRecord({
+                id: 1,
+                unique_id: "cp_incident_record_owner",
+                parent_form: "incident",
+                name: { en: "Form name" },
+                fields: [1]
+              }),
+              2: FormSectionRecord({
+                id: 2,
+                unique_id: "assessment",
+                parent_form: "case",
+                name: { en: "Assessment" },
+                fields: [2],
+                is_nested: false
+              }),
+              3: FormSectionRecord({
+                id: 3,
+                unique_id: "followup",
+                parent_form: "case",
+                name: { en: "followup" },
+                fields: [3],
+                is_nested: false
+              })
+            }),
+            fields: OrderedMap({
+              1: FieldRecord({
+                id: 1,
+                name: "test_service",
+                display_name: { en: "Test Field" },
+                type: "text_field",
+                multi_select: false,
+                form_section_id: 1,
+                visible: true,
+                mobile_visible: true
+              }),
+              2: FieldRecord({
+                id: 2,
+                name: "case_plan_due_date",
+                display_name: { en: "Case Plan Due Date" },
+                type: "text_field",
+                multi_select: false,
+                form_section_id: 2,
+                visible: true,
+                mobile_visible: true
+              }),
+              3: FieldRecord({
+                id: 3,
+                name: "test_follow_up",
+                display_name: { en: "Followup" },
+                type: "text_field",
+                multi_select: false,
+                form_section_id: 2,
+                visible: true,
+                mobile_visible: true
+              })
+            }),
+            options: {
+              lookups: [
+                {
+                  id: 1,
+                  unique_id: "lookup-service-type",
+                  values: [
+                    { id: "a", display_text: { en: "Service a" } },
+                    { id: "b", display_text: { en: "Service b" } }
+                  ]
+                }
+              ]
+            }
+          }
+        })
+      );
+    expect(screen.getAllByText(/record_id_display/i)).toHaveLength(4)
+    expect(screen.getAllByRole('row')).toHaveLength(5)
```

Comment:
This does not seem right. Looking at the old test we should be trying to find specific task types.

Reply:
Done